src/StrawberryShake/CodeGeneration/src/CodeGeneration.CSharp/CSharpGeneratorSettings.cs

```diff
@@ -61,6 +61,12 @@ public class CSharpGeneratorSettings
         public IDocumentHashProvider HashProvider { get; set; } =
             new Sha1DocumentHashProvider(HashFormat.Hex);
 
+        /// <summary>
+        /// Visibility overrides.
+        /// </summary>
+        public VisibilityOverrides VisibilityOverrides { get; set; }
```

Comment:
I think lets think a bit broader. We want to allow the user not only to override the access modifiers DI but for multiple things like state and so on. This means that we want to have more switches on this.

Reply:
OK, just saw the class .... closing this remark.